saleor/userprofile/i18n.py

```diff
@@ -37,6 +48,12 @@
     'zip': pgettext_lazy('Address field', 'ZIP code')}
 
 
+class PhonePrefixWidget(PhoneNumberPrefixWidget):
```

Comment:
I decided to overwrite this one as there are issues with default widget value:
https://github.com/stefanfoulis/django-phonenumber-field/issues/82
Quick example: choosing "United Kingdom +44 -> continue -> select new address makes field being populated by [Guernsey](https://en.wikipedia.org/wiki/Guernsey) as it has the same phone code, but it's alphabetically before UK.